avocado/utils/configure_network.py

```diff
@@ -90,11 +94,15 @@ def _write_to_file(self, filename, values):
             for key, value in values:
                 fp.write("{}={}\n".format(key, value))
 
-    def add_hwaddr(self, hwaddr):
-        """
-        Utility which add mac address to a interface return Status
+    def set_hwaddr(self, hwaddr):
+        """Set a Hardware Address (MAC Address) to the interface.
```

Comment:
please add the final docstrings in the implementation commit so we don't have to review them twice.